x-pack/filebeat/module/netflow/log/ingest/pipeline.yml

```diff
@@ -0,0 +1,54 @@
+---
+description: Pipeline for Filebeat NetFlow
+
+processors:
+  # IP Geolocation Lookup
+  - geoip:
+        if: ctx.source?.geo == null
+        field: source.ip
+        target_field: source.geo
+        ignore_missing: true
+  - geoip:
+        if: ctx.destination?.geo == null
+        field: destination.ip
+        target_field: destination.geo
+        ignore_missing: true
+
+  # IP Autonomous System (AS) Lookup
+  - geoip:
+        database_file: GeoLite2-ASN.mmdb
+        field: source.ip
+        target_field: source.as
+        properties:
+            - asn
+            - organization_name
+        ignore_missing: true
+  - geoip:
+        database_file: GeoLite2-ASN.mmdb
+        field: destination.ip
+        target_field: destination.as
+        properties:
+            - asn
+            - organization_name
+        ignore_missing: true
+  - rename:
+        field: source.as.asn
+        target_field: source.as.number
+        ignore_missing: true
+  - rename:
+        field: source.as.organization_name
+        target_field: source.as.organization.name
+        ignore_missing: true
+  - rename:
+        field: destination.as.asn
+        target_field: destination.as.number
+        ignore_missing: true
+  - rename:
+        field: destination.as.organization_name
+        target_field: destination.as.organization.name
+        ignore_missing: true
```

Comment:
Should these renames be added to breaking changes in changelog?

Reply:
No, those are new fields being added a few lines up. These rename are necessary to comply with ECS.

Reply:
Oh, I see :+1: